src/TLuaInterpreter.cpp

```diff
@@ -7129,13 +7130,13 @@ int TLuaInterpreter::setAreaName(lua_State* L)
 
     bool result = host.mpMap->mpRoomDB->setAreaName(id, newName);
     if (result) {
-        // Update mapper Area names widget, using method designed for it...!
```

Comment:
It might not be obvious to other coders so this is trying to explain why we are modifying `host.mpMap->mpMapper->showArea` which is the poorly named "mapper Area names widget"...

Reply:
If anyone runs into it being confusing, I'll put it back in. Otherwise comments should explain the 'why', not the 'what' - takes 2min to figure out what it does.